.github/workflows/build.yml

```diff
@@ -265,6 +265,20 @@ jobs:
           name: ${{ steps.diff.outputs.diff_name }}
           path: ${{ steps.diff.outputs.diff_name }}
 
+  aws-sdk-spec-test:
+    needs:
+      - build
+    if: github.event_name == 'push'
+    uses: ./.github/workflows/tf-aws-test.yml
+    secrets: inherit
+
+  file-diff-sdk-spec-test:
+    needs:
+      - build
+    if: github.event_name == 'pull_request'
```

Comment:
Is running this in PRs just for testing? I feel like we'll hit rate limits if we run this in every PR